mongodbatlas/fw_provider.go

```diff
@@ -354,6 +394,13 @@ func setDefaultValuesWithValidations(data *tfMongodbAtlasProviderModel, awsRoleD
 		}, "").(string))
 	}
 
+	if data.SecretName.ValueString() == "" {
```

Comment:
do we need yo check if we can't find the secret name?

Reply:
based on what we do with the other values, no